src/AutoMapper/Configuration/MemberConfigurationExpression.cs

```diff
@@ -284,9 +283,88 @@ public void SetMappingOrder(int mappingOrder)
             PropertyMapActions.Add(pm => pm.MappingOrder = mappingOrder);
         }
 
+        public void ConvertUsing<TValueConverter, TSourceMember>()
+            where TValueConverter : IValueConverter<TSourceMember, TMember>
+        {
+            PropertyMapActions.Add(pm =>
```

Comment:
You could have a common function that calls PropertyMapActions.Add and sets the config. Small, but useful :)

Reply:
Good call. Looks like there are many overloads in these classes that could use this refactoring.